File: app/src/main/java/com/tammamkhalaf/myuaeguide/categories/openTripMap/Info.java
package com.tammamkhalaf.myuaeguide.categories.openTripMap;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

public class Info {

    @SerializedName("descr")
    @Expose
    private String descr;
    @SerializedName("image")
    @Expose
    private String image;
    @SerializedName("img_width")
    @Expose
    private Integer imgWidth;
    @SerializedName("src")
    @Expose
    private String src;
    @SerializedName("src_id")
    @Expose
    private Integer srcId;
    @SerializedName("img_height")
    @Expose
    private Integer imgHeight;

    public String getDescr() {
        return descr;
    }

    public void setDescr(String descr) {
        this.descr = descr;
    }

    public String getImage() {
        return image;
    }

    public void setImage(String image) {
        this.image = image;
    }

    public Integer getImgWidth() {
        return imgWidth;
    }

    public void setImgWidth(Integer imgWidth) {
        this.imgWidth = imgWidth;
    }

    public String getSrc() {
        return src;
    }

    public void setSrc(String src) {
        this.src = src;
    }

    public Integer getSrcId() {
        return srcId;
    }

    public void setSrcId(Integer srcId) {
        this.srcId = srcId;
    }

    public Integer getImgHeight() {
        return imgHeight;
    }

    public void setImgHeight(Integer imgHeight) {
        this.imgHeight = imgHeight;
    }

}

File: shared/src/commonMain/kotlin/com/tammamkhalaf/kmmsharedmodule/categories/openTripMap/Info.kt
package com.tammamkhalaf.myuaeguide.categories.openTripMap

import com.google.gson.annotations.Expose

class Info {
    @SerializedName("descr")
    @Expose
    private var descr: String? = null

    @SerializedName("image")
    @Expose
    private var image: String? = null

    @SerializedName("img_width")
    @Expose
    private var imgWidth: Int? = null

    @SerializedName("src")
    @Expose
    private var src: String? = null

    @SerializedName("src_id")
    @Expose
    private var srcId: Int? = null

    @SerializedName("img_height")
    @Expose
    private var imgHeight: Int? = null
    fun getDescr(): String? {
        return descr
    }

    fun setDescr(descr: String?) {
        this.descr = descr
    }

    fun getImage(): String? {
        return image
    }

    fun setImage(image: String?) {
        this.image = image
    }

    fun getImgWidth(): Int? {
        return imgWidth
    }

    fun setImgWidth(imgWidth: Int?) {
        this.imgWidth = imgWidth
    }

    fun getSrc(): String? {
        return src
    }

    fun setSrc(src: String?) {
        this.src = src
    }

    fun getSrcId(): Int? {
        return srcId
    }

    fun setSrcId(srcId: Int?) {
        this.srcId = srcId
    }

    fun getImgHeight(): Int? {
        return imgHeight
    }

    fun setImgHeight(imgHeight: Int?) {
        this.imgHeight = imgHeight
    }
}